src/nvim/api/private/defs.h

```diff
@@ -38,7 +38,7 @@ typedef enum {
 
 typedef struct {
   ErrorType type;
-  char msg[1024];
+  char *msg;
   bool set;
```

Comment:
I usually put such data into `type` if present: if I was writing this struct there would be `kErrorTypeNone` and no flag.